pages/common/gh-pr.md

```diff
@@ -30,7 +30,7 @@
 
 - Edit the base branch of a pull request:
 
-`gh pr edit --base {{branch_name}}`
+`gh pr edit --base {{string}}`
```

Comment:
`branch_name` is 10x fast to understand here at the first read.
```suggestion
`gh pr edit --base {{branch_name}}`
```

Reply:
Hmm... I guess it can be formalized: use `{{string}}` when the option or subcommand name contains one noun describing expected argument, otherwise prefer non-generic placeholder.